source/_integrations/vallox.markdown

```diff
@@ -91,3 +91,12 @@ Set the fan speed of the `Boost` profile.
 | Service data attribute | Optional | Description                                     |
 |------------------------|----------|-------------------------------------------------|
 | `fan_speed`            |       no | Fan speed in %. `Integer`, between 0 and 100.   |
+
+### Action `vallox.set_profile` 
+
+Set the profile, and optionally a duration for the profile to be active.
+
+| Data attribute | Optional | Description                                                                                                                                                                          |
+|------------------------|---------:|--------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| `profile`              |       no | Profile to set, one of `Home`, `Away`, `Boost`, `Fireplace`, or `Extra`.                                                                                                              |
+| `duration`             |      yes | Duration to activate the profile for; in minutes. `Integer` between 1 and 65535. Only applies to  `Boost`, `Fireplace` or `Extra` profiles. 65535 sets the profile without a timeout. | 
```

Comment:
```suggestion
| `duration`             |      yes | Duration to activate the profile for; in minutes. `Integer` between 1 and 65535. Only applies to  `Boost`, `Fireplace` or `Extra` profiles. 65535 sets the profile without a timeout. | 
```
should we say that 65535 is about 45.5 days?

Reply:
65535 is actually a special case to set the profile permanently

Reply:
I see. Could you add a note to the description?

Reply:
it's the last part

the code changed a bit, i'll try to update this later today to match that and get this merged

Reply:
Oh dear. My oversight. Thank you.